src/include/ipc/topology.h

```diff
@@ -46,6 +46,7 @@ enum sof_comp_type {
 	SOF_COMP_DEMUX,
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
+	SOF_COMP_SMART_AMP,		/**< smart amplifier component */
```

Comment:
This one is blocking on kernel update, please check with @keyonjie

Reply:
@lgirdwood the kernel corresponding change is already merged:
https://github.com/thesofproject/linux/pull/1971